docs/CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Added
 -----
 - Custom signal ``resolwe.flow.signals.before_processing`` is sent when data
   object is ready for processing
+- Support setting ``descriptor`` / ``DescriptorSchema`` during process runtime
```

Comment:
Please make it clear this meants descriptor / DescriptorSchema on data objects (and not an Samples/Collections...).